src/scripts/tweaks/footer_unredesign.js

```diff
@@ -20,6 +20,10 @@ ${keyToCss('noteCount')} {
   gap: var(--post-padding);
 }
 
+${keyToCss('controls')} {
```

Comment:
`controls` maps to seven classes, so let's increase the specificity here

```suggestion
article footer > ${keyToCss('controls')} {
```

(should probably apply this to the noteCount rule above, too)

Reply:
Oh yikes, whoops. I was definitely thinking about that earlier and then completely forgot!